pulp_container/app/modelresource.py

```diff
@@ -0,0 +1,99 @@
+from import_export import fields, widgets
+from pulpcore.plugin.importexport import QueryModelResource, BaseContentResource
+
+from pulp_container.app.models import Blob, Manifest, ManifestListManifest, Tag
+
+
+class BlobResource(BaseContentResource):
+    """
+    Resource for import/export of blob entities
+    """
+
+    def set_up_queryset(self):
+        """
+        :return: Blobs specific to a specified repo-version.
+        """
+        return Blob.objects.filter(pk__in=self.repo_version.content)
```

Comment:
Add an explicit order-by to your filters. Without it, it's possible to end up with deadlocks at import-time - see https://github.com/pulp/pulp_rpm/pull/1975 for where we fixed pulp_rpm